cmd/commands/cli/command_identities.go

```diff
@@ -415,6 +415,9 @@ func (c *cliApp) getBeneficiaryStatus(actionArgs []string) error {
 		return fmt.Errorf("could not get beneficiary change status: %w", err)
 	}
 
+	if st.State == beneficiary.NotFound {
+		return nil
```

Comment:
Could we maybe do:
```suggestion
		clio.Info("No last change request information")
		return nil
```

or something like that